datahub-web-react/src/app/entity/dataJob/DataJobEntity.tsx

```diff
@@ -178,6 +196,7 @@ export class DataJobEntity implements Entity<DataJob> {
         return {
             urn: entity?.urn,
             name: entity?.properties?.name || '',
```

Comment:
and if we're gonna be consistent about using `displayName` we could also update this here to be `name: this.displayName(entity),`